matchms/data/known_key_conversions.csv

```diff
@@ -1,205 +1,206 @@
+ac$chromatography:column_temperature,column_temperature
+comment,comment
+comments,comment
+compound_type,compound_class
+ch$compound_class,compound_class
+ontology,compound_class
+name,compound_name
+compoundname,compound_name
+ch$name,compound_name
+comment:confidence,confidence
+libraryquality,confidence
+copyright,copyright
+datacollector,data_collector
+comment:datacollector,data_collector
+ms$data_processing:find_peak,data_processing_find_peak
+ms$data_processing:whole,data_processing_whole
+ms$focused_ion:derivative_form,derivative_formula
+ms$focused_ion:derivative_mass,derivative_mass
+ms$focused_ion:derivative_type,derivative_type
+ac$mass_spectrometry:desolvation_gas_flow,desolvation_gas
+ac$chromatogrpahy:drift_time,drift_time
+filename,file_name
+comment:filename,file_name
+ac$chromatography:flow_rate,flow_rate
+ch$formula,formula
+formula,formula
+ac$mass_spectrometry:fragmentation_mode,fragmentation_mode
+special_fragmentation,fragmentation_mode
+ac$mass_spectrometry:reagent_gas,gc_reagent_gas
+ac$chromatography:flow_gradient,gradient
+ch$iupac,inchi
+stdinchi,inchi
+inchi,inchi
+comment:inchiaux,inchi_aux
+inchiaux,inchi_aux
+ch$link:inchikey,inchikey
+inchikey,inchikey
+ac$mass_spectrometry:insource_voltage,insource_voltage
+cone_voltage,insource_voltage
+ac$instrument,instrument
+instrument,instrument
+ac$instrument_type,instrument_type
+instrument_type,instrument_type
+instrumenttype,instrument_type
+source_instrument,instrument_type
+ac$chromatogrpahy:inverse_reduced_mobility,inverse_reduced_mobility
+ac$mass_spectrometry:ion_mobility_drift_gas,ion_mobility_drift_gas
+ac$mass_spectrometry:ion_mobility_type,ion_mobility_type
+ac$mass_spectrometry:ion_mobility_voltage,ion_mobility_voltage
+ms$focused_ion:ion_type,ion_type
+ionimethod,ionization
+ac$mass_spectrometry:ionization,ionization
+ionization,ionization
+ac$mass_spectrometry:ionization_energy,ionization_energy
+ion_mode,ionmode
+ionpolarity,ionmode
+ionpol,ionmode
+ac$mass_spectrometry:ion_mode,ionmode
+ionmode,ionmode
+ms$focused_ion:isolation_window_lower_offset,isolation_window_lower_offset
+isolwidth,isolation_window_target_mz
+isolationwidth,isolation_window_target_mz
+isolation_width,isolation_window_target_mz
+ms$focused_ion:isolation_width,isolation_window_target_mz
+ms$focused_ion:isolation_window_upper_offset,isolation_window_upper_offset
+ch$link:kegg,kegg
+ch$link:knapsack,knapsack
+ac$mass_spectrometry:known_impurity,known_impurity
+known_impurity,known_impurity
+license,license
+ch$link:misc,link
+link,link
+ch$link:lipidbank,lipidbank
+ch$link:lipidmaps,lipidmaps
+ac$mass_spectrometry:laser,maldi_laser
+ac$mass_spectrometry:matrix,maldi_matrix
+ac$mass_spectrometry:mass_accuracy,mass_accuracy
+massrange,mass_range_mz
+massrangemz,mass_range_mz
+mass_range,mass_range_mz
+ac$mass_spectrometry:mass_range_m/z,mass_range_mz
+ac$mass_spectrometry:ms_type,ms_level
+spectrum_type,ms_level
+mslevel,ms_level
+ms_type,ms_level
```

Comment:
Not sure why this seems like everything was changed, but only ms_type, ms_level was added